ui/apps/platform/src/Components/PatternFly/DayPickerDropdown.tsx

```diff
@@ -97,7 +97,7 @@ function DayPickerDropdown({
         <Select
             variant={SelectVariant.checkbox}
             aria-label="Select one or more days"
-            onToggle={onToggleDaySelect}
+            onToggle={(_e, v) => onToggleDaySelect()}
```

Comment:
We're accepting a second, non-underscored param in this inline function, but why are we not using it?